app/controllers/v1/sessions_controller.rb

```diff
@@ -31,8 +31,9 @@ class SessionsController < ApplicationController
     # For more details see SAML::SSOeSettingsService and SAML::URLService
     def new
       type = params[:type]
-
-      if type == 'slo'
+      if Settings.saml_ssoe.mock
```

Comment:
I like the overall idea but I'm not sure if I like this being embedded in `sessions_controller`. What do you think about making a separate controller entirely for this, like `mock_auth_controller`? I think it'd have the following benefits:
* Accidentally enabling `Settings.saml_ssoe.mock` on any stack wouldn't impact the standard auth flow
* It would make mocked authentication a very intentional action by forcing devs to type in `localhost:3000/mock_auth`
* We could enable the route itself only on non-prod environments, preventing misuse of mocked auth, even if the settings were improperly set

Reply:
I agree with this approach from Trevor. This is too high of a security risk to be put in the same controller used for our current auth flows. Creating a separate controller entirely should reduce the risk and provide engineers with the ability to mock auth. @patrickvinograd what are your thoughts on this approach? Our team cannot approve this change as is, so we should collab on getting something that meets your requirements but also is safe.